galvan-support/src/main/java/org/terracotta/testing/config/TcConfigConvertor.java

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Can this be moved to tc-platform?  It doesn't seem to be used here.

Reply:
It's being used in `BasicExternalCluster`

Reply:
can you move this closer to tc-platform where the tool actually lives?

Reply:
Done